### PR TITLE
Global store mobx

### PR DESCRIPTION
**Pull request by dwalintukan**
- Renamed `PubsubStore` to `GlobalStore` for any globally used values
- This is for setting up `syncInfo` calls using MobX and storing it. Once the refactors for other components where they need the `syncInfo attributes: syncPercent, syncBlockNum, or syncBlockTime` you can use this `GlobalStore`'s values.

### CODE REVIEW

**Comment by amazingandyyy** [comment]
this will also relate to my connection-count PR. #713

File: src/appProvider.js [hunk 1]
```diff
@@ -20,7 +20,7 @@ export const AppProvider = observer(({ xstore, store }) => (
   <Provider store={store}>
     <MobxProvider store={xstore}>
       <MuiThemeProvider theme={bodhiTheme}>
-        <IntlProvider locale={xstore.ui.locale} messages={xstore.ui.localeMessages}>
+        <IntlProvider locale={xstore.global.locale} messages={xstore.global.localeMessages}>
```
Comment (alex-cory) [inline src/appProvider.js]: can we please leave the `PubSubStore` and the `UiStore` separate?
[UiStore Example](https://mobx.js.org/best/store.html)
image

The PubSubStore is used to modify other stores when a new block comes in. For instance, let's say we're on the `QTUM Prediction` page, and we get some new data, we then call `this.app.qtumPrediction.init()` from the `PubSubStore`. These are just 2 very separate concerns.

Comment (alex-cory) [reply src/appProvider.js]: We can also put `Error` in the `UiStore` since it's the `state of a global overlay`

Comment (alex-cory) [reply src/appProvider.js]: If we begin to have more and more realtime things happen with our app, this `GlobalStore` is going to get pretty long. I can show you an example from my last company if you'd like

**Pull request merged**